TESTING.md

```diff
@@ -0,0 +1,44 @@
+This cookbook uses a variety of testing components:
+
+- Unit tests: [ChefSpec](http://code.sethvargo.com/chefspec/)
```

Comment:
broken link.... probably want `https://docs.chef.io/chefspec.html`

Reply:
That's what I get for copying the file from another repo and just replacing the org/repo names.